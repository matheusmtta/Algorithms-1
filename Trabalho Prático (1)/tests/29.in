10 i
27 i
80 i
6 i
201 p
27 r
270 p
80 r
76 p
98 p
90 p
84 p
3 i
64 i
64 r
6 r
57 p
44 i
162 p
3 r
57 i
47 i
51 i
44 r
107 p
154 p
271 p
49 i
47 r
57 r
34 i
61 i
34 r
213 p
99 p
49 r
61 r
48 i
18 i
51 r
130 p
230 p
50 i
48 r
75 i
66 i
56 i
43 i
16 i
335 p
56 r
335 p
320 p
295 p
320 p
22 i
50 r
74 i
384 p
22 r
60 i
10 r
62 i
389 p
392 p
367 p
24 i
369 p
2 i
16 r
376 p
393 p
41 i
75 r
35 i
43 r
76 i
74 r
66 r
340 p
54 i
370 p
62 r
340 p
29 i
29 r
54 r
72 i
2 r
58 i
404 p
37 i
17 i
13 i
17 r
35 r
77 i
371 p
72 r
339 p
389 p
339 p
70 i
53 i
5 i
394 p
4 i
70 r
73 i
428 p
13 r
60 r
73 r
401 p
12 i
12 r
390 p
15 i
37 r
65 i
65 r
31 i
5 r
46 i
373 p
77 r
58 r
279 p
59 i
347 p
21 i
59 r
333 p
301 p
52 i
28 i
41 r
24 r
432 p
