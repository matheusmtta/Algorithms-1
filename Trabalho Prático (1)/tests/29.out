6
6
8
11
9
10
7
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
6
